camelAssemblies/src/main/resources/routeOverrides/fhir_sink/v4_4_3/fhir_sink.routes.yaml

```diff
@@ -0,0 +1,43 @@
+-   route-template:
```

Comment:
The following files are the override files we use for some assembly generations. Analogous to the v3_21_0 versions.